### PR TITLE
feat - added login and sign-up page

### PR DESCRIPTION
**Pull request by akash-1318**
https://user-images.githubusercontent.com/64393554/153706194-96abe04b-43ed-4cd0-a81b-340e8ed90b91.mov

Created Login and signup page : 
on Login page - Email, password, remember me checkbox, forgot Password and redirect link to Signup page
on Signup page - Email, password, checkbox for accepting conditions and redirect link to Login if user already has an account.

### CODE REVIEW

**Review by yogesh- — APPROVED** [review]
Looks good to merge.

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (darshandamre) [inline pages/login.html:7]: change the title

**Review by sahiba0915 — COMMENTED** [review]
Looks good to merge

**Review by darshandamre — APPROVED** [review]
looks good to me

File: pages/login.html (hunk 1 above)
Comment (darshandamre) [reply pages/login.html:7]: same in other files

**Pull request merged**